pom.xml

```diff
@@ -0,0 +1,228 @@
+    <groupId>io.lakefs</groupId>
+    <artifactId>hadoop-router-fs</artifactId>
+    <packaging>jar</packaging>
+    <version>0.1.0</version>
+
+    <licenses>
+        <license>
+            <name>apache2</name>
+            <url>http://www.apache.org/licenses/</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <name>Treeverse lakeFS dev</name>
+            <email>support@treeverse.io</email>
+            <organization>lakefs.io</organization>
+            <organizationUrl>https://lakefs.io</organizationUrl>
+        </developer>
+    </developers>
+
+    <scm>
+        <url>https://github.com/treeverse/hadoop-router-fs</url>
+        <connection>scm:git:git://github.com/treeverse/hadoop-router-fs.git</connection>
+        <developerConnection>scm:git:ssh://git@github.com/treeverse/hadoop-router-fs.git</developerConnection>
+    </scm>
+
+    <properties>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
+        <!--The common Hadoop version for Spark 2.4.7 and 3.0.1, which lakeFS supports-->
+        <hadoop.version>2.7.7</hadoop.version><!--MARKER: keep this comment here, we update the version with the lakeFS version when on deploy-->
+    </properties>
+
+    <profiles>
+        <profile>
+            <id>release</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-source-plugin</artifactId>
+                        <version>2.2.1</version>
+                        <executions>
+                            <execution>
+                                <id>attach-sources</id>
+                                <goals>
+                                    <goal>jar-no-fork</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-javadoc-plugin</artifactId>
+                        <version>3.3.1</version>
+                        <executions>
+                            <execution>
+                                <id>attach-javadocs</id>
+                                <goals>
+                                    <goal>jar</goal>
+                                </goals>
+                                <configuration>
+                                    <source>1.8</source>
+                                    <doclint>none</doclint>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+<!--        <profile>-->
```

Comment:
1. No code-in-comments, please.
1. If you do need this, just embed the *entire* element inside a *single* comment.  The only good thing about XML "comments" is that they nest.

Reply:
> 2. If you do need this, just embed the entire element inside a single comment. The only good thing about XML "comments" is that they nest.

Yikes!  They _don't_ nest, [so that XML will be compatible with SGML](https://www.w3.org/TR/2008/REC-xml-20081126/#sec-comments).  So I should revise this statement:
> 2. There is nothing good about XML comments.

So just ignore it, and I'd much rather get rid of the comments.

Reply:
@arielshaqed by "get rid of the comments" you mean to remove dead code altogether or uncomment them?

Reply:
I removed all the code in the comments and we will edit the pom file according to the implementation needs